src/protocols/memory/gddr7.stanza

```diff
@@ -0,0 +1,244 @@
+doc: \<DOC>
+@brief GDDR 7
+
+GDDR7 is a high speed memory protocol 
+@see https://en.wikipedia.org/wiki/GDDR7_SDRAM
+
+This functions and definitions in this file support defining GDDR7
+connections between microprocessors and memories in a board design.
+
+<DOC>
+#use-added-syntax(jitx)
+defpackage jsl/protocols/memory/gddr7:
+  import core
+  import jitx
+  import jitx/commands
+
+  import jsl/bundles
+  import jsl/errors
+  import jsl/ensure
+  import jsl/si
+  import jsl/pin-assignment
+
+doc: \<DOC>
+@brief GDDR7 Bundle
+One GDDR7 bundle consists of the connections between an integrated memory controller and 
+one GDDR7 memory chip. The overall memory connections consists of 4 separate channels.
+@member control Control channel for GDDR7
+@member data Four data channels for GDDR7
+<DOC>
+
+public defn gddr7 () :
+  gddr7-b()
+
+public pcb-bundle gddr7-b () :
+  name = "GDDR7"
+  description = "Graphics Double Data Rate 7 Synchronous Dynamic Random-Access Memory protocol"
+  port data    : gddr7-data-channel[4] ; one for each A B C D channel
+  port control : gddr7-control-channel ; covers all A,B,C,D
+
+public pcb-bundle gddr7-data-channel :
+  port DQ  : pin[10]   ; PAM3 or NRZ
+  port RCK : diff-pair ; NRZ
+  port WCK : diff-pair ; NRZ
+  port CA  : pin[5]    ; NRZ
+  port DQE             ; PAM3 or NRZ
+  port ERR
+
+public pcb-bundle gddr7-control-channel :
+  port RESET_n         ; NRZ
+  port ZQ_AB
+  port ZQ_CD
+
+
+doc: \<DOC>
```

Comment:
This docstring doesn't seem to be associated with anything. Is this right ?

Reply:
Perhaps it should go in the package level docstring ?

Reply:
Done